core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/dot-edit-content-wysiwyg-field.component.html

```diff
@@ -1 +1,3 @@
+@if (init()) {
 <editor [formControlName]="field.variable" [init]="init()" />
```

Comment:
`async` pipe better?

Reply:
The `init()` is a signal, not an observable

Reply:
I can make it an `observable` if you want